app/serializers/submission_serializer/sections/application_details.rb

```diff
@@ -13,7 +13,9 @@ def to_builder # rubocop:disable Metrics/AbcSize, Metrics/MethodLength
           json.date_stamp crime_application.date_stamp
           json.is_means_tested crime_application.is_means_tested
           json.ioj_passport crime_application.ioj_passport
-          json.means_passport crime_application.means_passport
+          json.means_passport(
```

Comment:
Dynamically calculating MeansPassport instead of relying on persisted value. It's hit and miss as to when this value was last persisted. 

I'd be in favour of removing persisting this value to the CrimeApplication altogether.